cypress/plugins/index.js

```diff
@@ -61,57 +60,63 @@ module.exports = (on, config) => {
 const MongoClient = require('mongodb').MongoClient
 const assert = require('assert')
 const tunnel = require('tunnel-ssh')
+const portScanner = require('portscanner')
 
 function bonnieFHIRDeleteMeasuresAndPatients (sshTunnel, config, userId) {
-  const sshTunnelConfig = {
-    agent: process.env.SSH_AUTH_SOCK,
-    username: sshTunnel.username,
-    privateKey: require('fs').readFileSync(sshTunnel.privateKey),
-    host: sshTunnel.host,
-    port: sshTunnel.port,
-    dstHost: sshTunnel.dstHost,
-    dstPort: sshTunnel.dstPort,
-    localHost: sshTunnel.localHost,
-    localPort: sshTunnel.localPort
-  }
-  console.log(sshTunnelConfig)
-  // tunnel -- See https://github.com/agebrock/tunnel-ssh#readme
-  tunnel(sshTunnelConfig, (error, server) => {
-    if (error) {
-      console.log("SSH connection error: ", error)
+  portScanner.findAPortNotInUse(50001, 60000, sshTunnel.localHost, function(error, port) {
+    console.log('AVAILABLE PORT AT: ' + port)
+
+    const sshTunnelConfig = {
+      agent: process.env.SSH_AUTH_SOCK,
+      username: sshTunnel.username,
+      privateKey: require('fs').readFileSync(sshTunnel.privateKey),
+      host: sshTunnel.host,
+      port: sshTunnel.port,
+      dstHost: sshTunnel.dstHost,
+      dstPort: sshTunnel.dstPort,
+      localHost: sshTunnel.localHost,
+      localPort: port
     }
-    // Connection URL
-    const url = 'mongodb://' + sshTunnelConfig.localHost + ':' + sshTunnelConfig.localPort
-    // Database Name
-    const dbName = config.env.mongo_db
-    // Use connect method to connect to the server
-    MongoClient.connect(url, { autoReconnect: true},function (err, client) {
-      assert.equal(null, err)
-
-      console.log("Connected successfully to server")
-
-      const db = client.db(dbName)
-      const {ObjectId} = require('mongodb')
-
-      //Delete patients based on User ID
-      db.collection('cqm_patients').removeMany({user_id: ObjectId(userId)}, (err, item) => {
-        if (err) {
-          console.log(err)
-        }
-        console.log(item)
-      })
+    console.log(sshTunnelConfig)
```

Comment:
I think this would spit out credentials in plain text. Not sure if this is any harm.

Reply:
Looks like it is logging ok. 
![image](https://user-images.githubusercontent.com/57112657/112051408-47884e00-8b28-11eb-8d92-6764894476fc.png)

Reply:
oh no, what I meant is this would reveal sensitive data in plain text.

Reply:
I should have been more clear there. Now your screenshot shows ip, and username 😬

Reply:
ok, I see what you are saying. I deleted that comment. But you wouldn't see anything unless you already have your env variables setup. I can remove it though. I would need to change the log for the delete commands too. Let me clean this up.

Reply:
@adongare I think this should be good now.